services/libs/tinybird/datasources/integrations.datasource

```diff
@@ -0,0 +1,13 @@
+SCHEMA >
+    `id` String `json:$.record.id`,
+    `platform` String `json:$.record.platform` DEFAULT '',
+    `status` String `json:$.record.platform` DEFAULT '',
+    `settings` String `json:$.record.settings` DEFAULT '',
+    `integrationIdentifier` String `json:$.record.integrationIdentifier` DEFAULT '',
+    `segmentId` String `json:$.record.settings` DEFAULT '',
```

Comment:
Same question as above. This seems to reference an incorrect entity. But I'm not very familiar with our schemas, so I'm not sure.